sleep_for_days/workflows.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+from datetime import timedelta
+from temporalio import workflow
+from dataclasses import dataclass
+from sleep_for_days.activities import send_email, SendEmailInput
+
+@dataclass
+class SleepForDaysInput:
+    numOfDays: int
+
+@workflow.defn(name="SleepForDaysWorkflow")
+class SleepForDaysWorkflow:
+
+    def __init__(self) -> None:
+        self.is_complete = False
+
+    @workflow.run
+    async def run(self, input: SleepForDaysInput) -> str:
+        while(not self.is_complete):
+            await workflow.execute_activity(send_email, SendEmailInput(f"{input.numOfDays} until the next email"), start_to_close_timeout=timedelta(seconds=10))
+            await workflow.wait([
+                asyncio.create_task(workflow.sleep(input.numOfDays * 24 * 60 * 60)),
```

Comment:
This would look nicer as a `timedelta` which `sleep` accepts rather than doing all this multiplication